tests/ci/codebuild/linux-aarch/run_minimal_tests.yml

```diff
@@ -8,7 +8,6 @@ phases:
     commands:
       # To use this build spec file, CMake environment variable CC and CXX compiler should be defined before build.
       - if [[ -z "${CC+x}" || -z "${CC}" ]]; then echo "CC is not defined." && exit 1; else ${CC} --version && echo "Found correct CC."; fi
-      - if [[ -z "${CXX+x}" || -z "${CXX}" ]]; then echo "CXX is not defined." && exit 1; else ${CXX} --version && echo "Found correct CXX."; fi
```

Comment:
Why is this line removed? If it's not needed, should the comment on l.9 be adjusted?

Reply:
Added it back.